src/test/javascript/spec/component/shared/metis/postings-create-edit-modal/post-create-edit-modal/post-tag-selector.component.spec.ts

```diff
@@ -26,7 +26,7 @@ describe('PostTagSelectorComponent', () => {
             imports: [],
             providers: [{ provide: MetisService, useClass: MockMetisService }],
             declarations: [PostTagSelectorComponent, MockPipe(ArtemisTranslatePipe)],
-            schemas: [NO_ERRORS_SCHEMA],
+            schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
This is still using `CUSTOM_ELEMENTS_SCHEMA`?

Reply:
Nope, I've overlocked it. I changed it in a549ab0